public/jsx/components/product-detail.jsx

```diff
@@ -17,6 +17,11 @@ module.exports = React.createClass({
     var productId = this.props.productId;
     var productData = products[productId];
 
+    var logo = document.getElementById('logo');
+    if (logo) {
+        logo.style.backgroundImage = 'url(' + productData.img + ')';
+    }
```

Comment:
Altering the DOM directly is a no-no in React because you lose its reactive properties.

What you can do instead (I think) is render the app view with your div element as a wrapper around the ``<RouteHandler>``. In [app.jsx](https://github.com/andymckay/payments-ui/blob/transparent-background/public/jsx/app.jsx#L31) I think you can make render look more like this:
````
render () {
  var productData = products[this.state.productId];
  return (
    <div id="logo" style="background-image: url({productData.img})"></div>
    <main>
      <div id="view">
        <RouteHandler productId={this.state.productId} />
      </div>
    </main>
  );
},
````